### PR TITLE
Article Page Layout

### PR DESCRIPTION
**Pull request by andemerie**
Selection_003
Selection_004

### CODE REVIEW

File: pages/article.js [hunk 1]
```diff
@@ -37,17 +45,65 @@ class ArticlePage extends Component {
 
   render() {
     const { article } = this.props;
-    const { articleId, slug, title, description } = article;
+    const { description, covers, tags, title, subtitle, imagePreviewUrl } = article;
```
Comment (Drapegnik) [inline pages/article.js]:
```suggestion
    const { article: { description, covers, tags, title, subtitle, imagePreviewUrl } } = this.props;
```

**Review by Drapegnik — CHANGES_REQUESTED** [review]
Could you, please, remove `pages/legacy/article` and unused `PublicArticle` components?

File: pages/article.js [hunk 2]
```diff
@@ -37,17 +45,65 @@ class ArticlePage extends Component {
...
+                <span>{formatNodeList(authors.map(tag => getTagLink({ tag })))}</span>
+              </div>
+            </div>
+          )}
+          <div className="article-page__title">{title}</div>
+        </div>
+        <div className="article-page-margins">
+          <div>There should be content from editor</div>
+          <div className="article-page__share">
+            <ShareButtons />
```
Comment (Drapegnik) [inline pages/article.js]: please, implement **sharing**, like in `ShareToButton.js`

File: pages/article.js [hunk 3]
```diff
@@ -37,17 +45,65 @@ class ArticlePage extends Component {
...
+          <div className="article-page__other-tags">
+            {Object.values(tagLists).map(tagList =>
+              tagList.map(tag => (
+                <div key={tag.slug} className="article-page__other-tag">
+                  {getTagLink({ tag })}
+                </div>
+              ))
+            )}
+          </div>
+        </div>
```
Comment (Drapegnik) [inline pages/article.js:108]: what's about related articles block?

**Comment by Drapegnik** [comment]
[admin: articles page](https://trello.com/c/I1v3J11m/369-admin-articles-page)

File: constants/social.js [hunk 4]
```diff
@@ -41,15 +39,29 @@ export const YOUTUBE_EMBED_PREFIX = 'https://www.youtube.com/embed';
...
+    getParams: (url, text) => ({ url, text }),
+  },
+  {
+    id: 'vk',
+    baseUrl: 'https://vk.com/share.php',
+    getParams: (url, title) => ({ url, title }),
+  },
+  {
+    id: 'odnoklassniki',
+    icon: 'odnoklassniki',
```
Comment (Drapegnik) [inline constants/social.js]: redundant

**Review by Drapegnik — CHANGES_REQUESTED** [review]
and, please, fix lint errors

**Review by Drapegnik — APPROVED** [review]

**Comment by andemerie** [comment]
[article page frontend](https://trello.com/c/uTQAYI3T/379-article-page-frontend)

**Pull request merged**